### ISSUE TITLE
When app runs on CXL memory, the `train` of Liblinear runes extremely slow (By Yan)

### ISSUE THREAD

**Issue by yulistic** [yulistic/multi-clock#3]
Although binding works with cgroup -- I ran with this script:

```
/home/yans3/cxlmem_test_software/script/run_test/run_liblinear.sh
```

When app runs on CXL memory (for the first 120 second), the train process runes extremely slow -- maybe it is blocked by multi-clock kernel tasks (observed from top)

Once after 120 second, cgroup cpuset constrain is release, and memory is allocated to node 0, which makes the train finish properly.

**Comment by yulistic** [comment]
Need to check whether it is fixed by 3d4a4558dee964d08084c2a8526d1ce8752598d9.

**Comment by yulistic** [comment]
Fixed.
